src/Host.h

```diff
@@ -666,6 +666,7 @@ class Host : public QObject
     QMap<QString, QKeySequence*> profileShortcuts;
 
     bool mTutorialForCompactLineAlreadyShown;
+    bool mTutorialForSplitscreenScrollbackAlreadyShown;
```

Comment:
We are switching over to initialisation in the header file for those member that can be done like that - it is just that I haven't got around to this class yet. However it is less work all around if new things are done in the "new" way from the start so do this:
```suggestion
    bool mTutorialForSplitscreenScrollbackAlreadyShown = false;
```